strophejingle/index.html

```diff
@@ -14,7 +14,7 @@
       <form id="loginInfo">
         <label>JID: <input id="jid" type="text" name="jid" placeholder="me@example.com"/></label>
         <label>Password: <input id="password" type="password" name="password" placeholder="secret"/></label>
-        <label>BOSH URL: <input id="boshURL" type="text" name="boshURL" placeholder="/http-bind"/></label>
+        <label>BOSH URL: <input id="boshURL" type="text" name="boshURL" value="/http-bind/"/></label>
```

Comment:
I understand. With scripts that I've produced I tend to push this data to localStorage and then on the site populate the fields if that data is present. @legastero might not be against a PR for it :)